guides/doc-Administering_Red_Hat_Satellite/topics/Logging_in_to_Red_Hat_Satellite.adoc

```diff
@@ -90,7 +90,6 @@ This includes Content Views, Activation Keys, and Life Cycle Environments.
 | *Hosts*  | Provides host inventory and provisioning configuration tools.
 | *Configure*  | Provides general configuration tools and data including Host Groups and Puppet data.
 | *Infrastructure*  | Provides tools on configuring how {ProjectX} interacts with the environment.
-| *Insights*  | Provides Red{nbsp}Hat Insights management tools.
```

Comment:
What happened to RH Insights plugin? @ares I only see it mentioned in the issue.

Reply:
RH Insights plugin is removed at 6.10.

Reply:
The old plugin, it's replaced by rh cloud plugin

Reply:
We need to check if the Insights tab is still in place, Marek says that it has been replaced, not removed completely

Reply:
The Insights tab is referred in the below changes in this PR

Reply:
From SAT-4179
The following items deprecated in 6.9 should be dropped 6.10:
   Red Hat Access Insights plugin 

Comment on SAT-3192 by Mike McCune
The functionality in Satellite that was available in the top level navigation under the Insights menu options has now been directly integrated into the Hosts -> All Hosts user interface. There is no longer a separate Insights UI as we work to unify the experience of recommending and remediating actions for the hosts managed in Satellite.

Reply:
@spetrosi Does this answer the questions? Can we get this merged?

Reply:
Yup, thanks for the clarification, it's clear now. I do not have the permissions to merge PRs.